src/PhpDoc/ResolvedPhpDocBlock.php

```diff
@@ -202,6 +204,7 @@ public function merge(array $parents, array $parentPhpDocBlocks): self
 			}
 		}
 		$result->phpDocNodes = $phpDocNodes;
+		$result->phpDocString = $this->phpDocString;
```

Comment:
without this line I got errors like 
>Error: Typed property PHPStan\PhpDoc\ResolvedPhpDocBlock::$phpDocString must not be accessed before initialization

now I see line 197  mentioning `// skip $result->phpDocString - just for stubs` ..hmm?